SYCL/InorderQueue/in_order_kernels.cpp

```diff
@@ -1,13 +1,12 @@
-// UNSUPPORTED: cuda
-// CUDA does not support unnamed lambdas.
 //
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -fsycl-unnamed-lambda %s -o %t.out
 // RUN: %HOST_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 
-//==------ oq_kernels.cpp - SYCL ordered queue kernel shortcut test --------==//
+//==------ InorderQueue.cpp - SYCL ordered queue kernel shortcut test
```

Comment:
Minor comment: why "InorderQueue.cpp"? The test name is in_order_kernels.cpp.

The comment could be without the test name at all, something like: // This test checks the kernel call shortcut methods of SYCL ordered queue"

Reply:
Yes that was a mistake it should have read 'in_order_kernels.cpp'.  I'll switch it to your suggestion. Thanks